include/odp/api/spec/classification.h

```diff
@@ -320,7 +320,8 @@ int odp_cls_capability(odp_cls_capability_t *capability);
  * and pool associated with a class of service and when any one of these values
  * are configured as INVALID then the packets assigned to the CoS gets dropped.
  */
-odp_cos_t odp_cls_cos_create(const char *name, odp_cls_cos_param_t *param);
+odp_cos_t odp_cls_cos_create(const char *name,
```

Comment:
Traffic manager has also a few potential functions missing const qualifiers:
```
odp_tm_shaper_create()
odp_tm_shaper_params_update()
odp_tm_sched_create()
odp_tm_sched_params_update()
odp_tm_threshold_create()
odp_tm_thresholds_params_update()
odp_tm_wred_create()
odp_tm_wred_params_update()
odp_tm_node_create()
odp_tm_queue_create()
```

Reply:
All fixed in v2